src/coprocessor/codec/mysql/decimal.rs

```diff
@@ -2172,11 +2172,19 @@ impl<'a, 'b> Mul<&'a Decimal> for &'b Decimal {
     }
 }
 
+impl<'a, 'b> Div<&'a Decimal> for &'b Decimal {
+    type Output = Option<Res<Decimal>>;
+
+    fn div(self, rhs: &'a Decimal) -> Self::Output {
+        <Decimal>::div(self, rhs, DEFAULT_DIV_FRAC_INCR)
+    }
+}
+
 impl Div for Decimal {
```

Comment:
Maybe you can just remove this impl. `impl<'a, 'b> Div<&'a Decimal> for &'b Decimal {` is enough